tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -169,7 +169,7 @@
         Field(
             item="70A",
             name="RACE_HISPANIC",
-            friendly_name="race hispanic",
+            friendly_name="Ethnicity/Race: Hispanic or Latino",
```

Comment:
@victoriaatraft here you add "ethnicity/race:" but not in For M2, M3, T2, T3, TANF T2, or TANF T3. Please inform what the expected language should be. cc: @elipe17

Reply:
Thank you. I'll updated to remove "ethnicity/race"